docker-compose.yml

```diff
@@ -126,6 +127,11 @@ services:
       # If you have high volume and your search return incomplete results
       # You might want to change this to a higher value (and ensure your host has enough memory)
       MAX_MEMORY_USAGE_RATIO: 0.3
+  geoipupdate:
+    image: 'maxmindinc/geoipupdate:latest'
+    entrypoint: ['/usr/bin/geoipupdate', '-d', '/sentry', '-f', '/sentry/GeoIP.conf']
```

Comment:
We're overriding the entrypoint to avoid the [envvar shenanigans](https://github.com/maxmind/geoipupdate/blob/v4.5.0/docker/entry.sh#L9-L20), not to change the location of the mmdb file.

Reply:
By default they look for the conf file and mmdb file in different directories, so we need to do _something_ to wire up between the host and the geoipupdate container. This seems a fine way imo.